page-objects/pages/SubscribePage.js

```diff
@@ -1,47 +1,46 @@
-class Subscribe{
-    get emailAddress() {
-        return $('#MERGE0')
-    }
-
-    get firstName() {
-        return $('#MERGE1')
-    }
-
-    get lastName() {
-        return $('#MERGE2')
-    }
-
-    get submitButton() {
-        return $('.formEmailButton') 
-    }
-
-    get addButton() {
-        return $('.hcard-download')
-    }
-
-    get continueButton() {
-        return $$('.formEmailButton')[0]
-    }
-
-    get cmanageButton() {
-        return $$('.formEmailButton')[1]
-    }
-
-    get emailMeALinkButton(){
-        return $('.formEmailButton')
-    }
-
-    get error() {
-        return $('.formstatus error')
-    }
-
-    email(){
-        return Math.random().toString(36).substring(2,11) + '@domain.com';
-    }
-
-    get confirmSubscribe(){
-        return $('#templateBody')
-    }
-
+class Subscribe {
+  get emailAddress() {
+    return $("#MERGE0");
+  }
+
+  get firstName() {
+    return $("#MERGE1");
+  }
+
+  get lastName() {
+    return $("#MERGE2");
+  }
+
+  get submitButton() {
+    return $(".formEmailButton");
+  }
+
+  get addButton() {
+    return $(".hcard-download");
+  }
+
+  get continueButton() {
+    return $$(".formEmailButton")[0];
+  }
+
+  get cmanageButton() {
+    return $$(".formEmailButton")[1];
+  }
+
+  get emailMeALinkButton() {
+    return $(".formEmailButton");
+  }
+
+  get error() {
+    return $(".formstatus error");
+  }
+
+  email() {
```

Comment:
This method is generating a valid email. maybe you should be more suggesting with the name of the function.